app/main.py

```diff
@@ -1 +1,20 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import VaccineError, NotWearingMaskError
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
+    all_vaccinated = True
+    masks_to_buy = 0
+    for friend in friends:
+        try:
+            cafe.visit_cafe(friend)
+        except VaccineError:
+            all_vaccinated = False
+        except NotWearingMaskError:
+            masks_to_buy += 1
+    if not all_vaccinated:
+        return "All friends should be vaccinated"
```

Comment:
u don't need all_vaccinated variable
just return "All friends should be vaccinated" if VaccineError will be risen